pokemon_search/src/routes/details.js

```diff
@@ -0,0 +1,65 @@
+import { useParams, Link } from "react-router-dom";
+import { useState, useEffect } from "react";
+
+function Details() {
+  const [data, setData] = useState({});
+  const [error, setError] = useState(null);
+  const [loading, setLoading] = useState(false);
+  const params = useParams();
+
+  useEffect(() => {
+    async function fetchPokemon() {
+      try {
+        setLoading(true);
+        const response = await fetch(
+          `https://pokeapi.co/api/v2/pokemon/${params.pokemonId}`
+        );
+        const pokemonData = await response.json();
+        setData(pokemonData);
+        setLoading(false);
+      } catch (err) {
+        setError(err.message);
+      }
+    }
+    fetchPokemon();
+  }, []);
```

Comment:
Would our `useFetch` work to simplify this fetching logic? 🙂

Reply:
Foolish me! lol Great point!

Reply:
Do I have to export useFetch from App.js and import to details.js?